.gitignore

```diff
@@ -59,3 +59,7 @@ war/node_modules/
 war/yarn-error.log
 .java-version
 .checkstyle
+
+# ANTLR generated files
+*.tokens
+core/gen/
```

Comment:
I fail to see the need for these additions. I ran `git clean -fxd` and `mvn clean verify -DskipTests`, after which the only `.tokens` files were in `./core/target/classes/` and `./core/target/generated-sources/antlr4/` (which are already ignored by the first line of this file) and there were no `core/gen/` directory. I then imported the project, from scratch, into IntelliJ and ran the **Build Project** function in IntelliJ, with the same result.

```suggestion
```